clintrials/dosefinding/efftox.py

```
@@ -507,52 +544,92 @@ def intersection_expression(x, m, f):
+            # a + b/x_00 + c/x_00^2 = y_00
+            # (y_00 - a)x_00^2 - b*x_00 - c = 0
+            A = y_00 - self.a
             B = -self.b
             C = -self.c
 
             if A == 0:
                 if B == 0:
-                    return np.nan
+                    raise ValueError(
+                        f"Utility delta {delta} is infeasible for prob_tox={prob_tox}"
+                    )
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)